src/System.Collections/tests/Resources/System.Collections.Tests.rd.xml

```diff
@@ -1,7 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
 <Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
-  <Library>
+  <Library Name="System.Collections.Tests">
+    <Assembly Name="*Application*" DoNotInline="true" DoNotOptimize="true" />
```

Comment:
Also I added this rd.xml before we started using the ILCBuildType pattern. If @ViktorHofer wants to merge this now I can update it since I'm tracking this in my backlog.